app/src/main/java/com/example/mycleaninglog/dto/CleaningTask.kt

```diff
@@ -2,7 +2,7 @@ package com.example.mycleaninglog.dto
 
 import com.google.firebase.firestore.Exclude
 
-data class cleaningTask(var cleaningTaskName:String = "", var cleaningTaskId:String = ""){
+data class CleaningTask(var cleaningTaskName:String = "", var cleaningTaskId:String = ""){
```

Comment:
Changed class name to UpperCamelCase in line with Kotlin coding conventions. 
Here's a link to the reference sit I used: 
https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs

Reply:
Yes, good.